### ISSUE TITLE
Flaky endtoend test tabletmanager

### ISSUE THREAD

**Issue by sougou** [vitessio/vitess#6028]
This has started happening very often

```
2020/04/07 22:23:42 Output directory: _test/20200407-222342.10470
2020/04/07 22:23:42 Running make build...
2020/04/07 22:25:44 local.tabletmanager_zk2: running (try 1/3)...
running tests for  vitess.io/vitess/go/test/endtoend/tabletmanager --topo-flavor=zk2
I0407 22:25:53.477862   15937 cluster_process.go:580] Attempting to reserve port: 16001
I0407 22:25:53.478580   15937 cluster_process.go:588] Port 16001 is available, reserving..
I0407 22:25:53.478687   15937 cluster_process.go:580] Attempting to reserve port: 16002
I0407 22:25:53.478714   15937 cluster_process.go:588] Port 16002 is available, reserving..
I0407 22:25:53.478728   15937 cluster_process.go:580] Attempting to reserve port: 16003
I0407 22:25:53.478741   15937 cluster_process.go:588] Port 16003 is available, reserving..
I0407 22:25:53.478753   15937 cluster_process.go:580] Attempting to reserve port: 16004
I0407 22:25:53.478765   15937 cluster_process.go:588] Port 16004 is available, reserving..
I0407 22:25:53.478780   15937 cluster_process.go:175] Starting topo server zk2 on port: 16002
I0407 22:25:53.478785   15937 cluster_process.go:580] Attempting to reserve port: 16005
I0407 22:25:53.478797   15937 cluster_process.go:588] Port 16005 is available, reserving..
I0407 22:25:53.478807   15937 cluster_process.go:580] Attempting to reserve port: 16006
I0407 22:25:53.478819   15937 cluster_process.go:588] Port 16006 is available, reserving..
I0407 22:25:53.478851   15937 topo_process.go:140] Starting zookeeper with args zkctl -log_dir /home/runner/work/vitess/vitess/vtdataroot/vt_066308622/vtroot_16001/topo_16002/logs -zk.cfg 1@fv-az88:16005:16006:16002 init
E0407 22:25:53.528245   15937 cluster_process.go:177] exit status 2
E0407 22:25:53.537786   15937 cluster_process.go:531] Error in topo server teardown: exit status 2
FAIL	vitess.io/vitess/go/test/endtoend/tabletmanager	0.067s
FAIL
```

**Comment by arindamnayak** [comment]
I have moved that test to run in docker in #6025, the testcase is passing there.

**Comment by deepthi** [comment]
Fixed in #6025
